api/src/certification/enrolment/domain/models/Subscription.js

```diff
@@ -10,12 +10,12 @@ class Subscription {
    * @param {Object} params
    * @param {number} params.certificationCandidateId - identifier of the certification candidate
    * @param {SUBSCRIPTION_TYPES} params.type
-   * @param {number} params.complementaryCertificationId
+   * @param {string} params.complementaryCertificationKey
```

Comment:
(+ ajouter un import jsdoc le cas echeant)
```suggestion
   * @param {ComplementaryCertificationKeys} params.complementaryCertificationKey
```